### PR TITLE
feat(user_profile_form): Add photo size validation

### PR DESCRIPTION
**Pull request by josix**
## Types of changes
**Thanks for sending a pull request! Please fill in the following content to let us know better about this change.**
Please put an `x` in the box that applies

- [ ] **Bugfix**
- [x] **New feature**
- [ ] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [ ] **Other (please describe)**

## Description
**Describe what the change is**
- Add photo size validation in profile update form

## Steps to Test This Pull Request
1. Go to user profile page
2. Upload 8 x 8 image that is larger than 10M.

## Related Issue
If applicable, refernce to the issue related to this pull request.
https://trello.com/c/j0ezAODh/24-fix-all-the-bugs-found-during-the-proposal-system-testing

## More Information
**Screenshots**
image

**Additional context**
Add any other context about the problem here. You may also want to refer
to [how to wirte the perfect pull request](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)

### CODE REVIEW

File: src/locale/zh_Hant/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -4142,6 +4142,10 @@ msgstr "提供的圖片過小（{width}\\u00d7{height} 像素）。"
 msgid "The image you provided is not quadrate."
 msgstr "提供的圖片並非方型。"
 
+#: users/forms.py:
```
Comment (mattwang44) [inline src/locale/zh_Hant/LC_MESSAGES/django.po:4145]: nit: Please also add this message to `src/locale/en-US/LC_MESSAGES/django.po`.

**Review by mattwang44 — APPROVED** [review]
LGTM

**Comment by mattwang44** [comment]
[Fix ALL the bugs found during the proposal system testing](https://trello.com/c/j0ezAODh/24-fix-all-the-bugs-found-during-the-proposal-system-testing)

File: src/locale/zh_Hant/LC_MESSAGES/django.po (hunk 1 above)
Comment (josix) [reply src/locale/zh_Hant/LC_MESSAGES/django.po]: Hello @mattwang44, thanks for your remind. I've added it. Please have a check.

Comment (mattwang44) [reply src/locale/zh_Hant/LC_MESSAGES/django.po]: NP. Thanks for the efforts!

**Pull request merged**